src/omv/sensors/pag7920.c

```diff
@@ -0,0 +1,521 @@
+/*
+ * This file is part of the OpenMV project.
+ *
+ * Copyright (c) 2023 Lake Fu <lake_fu@pixart.com> for PixArt Inc.
+ *
+ * This work is licensed under the MIT license, see the file LICENSE for details.
+ *
+ * PAG7920 driver.
+ */
+#include "omv_boardconfig.h"
+#if (OMV_PAG7920_ENABLE == 1)
+
+#include <stdint.h>
+#include <stdio.h>
+#include <math.h>
+#include "py/mphal.h"
+
+#include "sensor.h"
+#include "framebuffer.h"
+#include "omv_i2c.h"
+
+#include "pag7920.h"
+#include "pag7920_reg.h"
+
+#define SENSOR_I2C_DEBUG (0)
+
+#define US (1000000)  // microsecond
+#ifndef M_LN29
+#define M_LN29  (3.36729582998647402718)
+#endif
+
+typedef union {
+    struct {
+        uint8_t b0 : 1;
+        uint8_t b1 : 1;
+        uint8_t b2 : 1;
+        uint8_t b3 : 1;
+        uint8_t b4 : 1;
+        uint8_t b5 : 1;
+        uint8_t b6 : 1;
+        uint8_t b7 : 1;
+    } bits;
+    uint8_t byte_val;
+} bitplane;
+
+extern uint8_t _line_buf;
+static int8_t g_bank_cache = -1;
+
+static bool g_f_hflip = false;
+static bool g_f_vflip = false;
+
+static int switch_bank(sensor_t *sensor, uint8_t bank) {
+    if (g_bank_cache != bank) {
+#if (SENSOR_I2C_DEBUG == 1)
```

Comment:
```suggestion
    #if (SENSOR_I2C_DEBUG == 1)
```
Those should match the indent of the following line, our formatter doesn't automatically fix it.